docs/src/userguide/source_files_and_compilation.rst

```diff
@@ -1064,7 +1064,11 @@ Cython code.  Here is the list of currently supported directives:
     tracing, unless you additionally pass the C macro definition
     ``CYTHON_TRACE=1`` to the C compiler (e.g. using the setuptools option
     ``define_macros``).  Define ``CYTHON_TRACE_NOGIL=1`` to also include
-    ``nogil`` functions and sections.
+    ``nogil`` functions and sections.  Define ``CYTHON_USE_SYS_MONITORING``
+    to either 1 or 0 to control the mechanism used to implement these
+    features on Python 3.13 and above.  Note that neither ``profile``
+    or ``linetrace`` work on Python 3.12 (but they do work on both
+    higher and lower versions).
```

Comment:
The mix of what works and what doesn't seems a bit more complex than that (I [just updated](https://github.com/cython/cython/commit/0ab0e32a19a3bfd917395da92a82f7e784c77cb2) the test selection to reflect that), but I think it's fair to put the user expectation regarding Py3.12 as low as possible.

Reply:
I've changed the wording a bit - I think it's basically that anything that's been updated to use `sys.monitoring` won't work, but anything using the old system might.